pkg/cosign/attestation/attestation.go

```diff
@@ -34,6 +34,9 @@ const (
 
 	// CosignVulnProvenanceV01 specifies the type of VulnerabilityScan Predicate
 	CosignVulnProvenanceV01 = "cosign.sigstore.dev/attestation/vuln/v1"
+
+	// PredicateCycloneDX represents a SBOM using the CycloneDX standard.
+	PredicateCycloneDX = "https://cyclonedx.org/Document"
```

Comment:
Was not sure the proper value for this. This is based on `https://spdx.dev/Document` defined within in-toto: https://github.com/in-toto/in-toto-golang/blob/af1f9fb822bf841779c916a954f651e908c5a818/in_toto/model.go#L81

Reply:
Filed this one on in-toto go, we could use the same value here: https://github.com/in-toto/in-toto-golang/pull/169

Reply:
Saw your comment there. Just changed to `/schema`